<?xml version="1.0"?>
<launch>
  <arg name="config_path" default="$(find dnn_detector)/config/dnn.yaml"/>

  <!-- ball detector -->
  <node pkg="dnn_detector" type="dnn_detector_node" name="dnn_detector_node" args="" output="screen">
    <rosparam command="load" file="$(arg config_path)"/>
    <param name="yaml_path" type="string" value="$(arg config_path)"/>
    <param name="proto"             value="$(find dnn_detector)/config/SoccerBall-Alfarobi_deploy.prototxt"/>
    <param name="model"             value="$(find dnn_detector)/config/SoccerBall-Alfarobi_deploy.caffemodel"/>
    <remap from="/dnn_detector_node/image_in" to="/alfarobi/camera/image_raw" />
    <param name="threshold"           value="0.6" type="double"/>
  </node>
</launch>

